src/main/java/in/projecteka/consentmanager/link/link/model/TokenConfirmation.java

```diff
@@ -0,0 +1,11 @@
+package in.projecteka.consentmanager.link.link.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@AllArgsConstructor
+@Data
```

Comment:
Can you use @value annotation?

Reply:
Typically Jackson would use reflection and use javabeans properties std (setter) to bind from JSON to class. @Value didn't play well, but seems to work now.

Reply:
Yeah, configuration is added to make it work :)